bodhi/server/services/releases.py

```diff
@@ -332,3 +350,47 @@ def save_release(request):
     request.db.flush()
 
     return r
+
+
+@critpath_packages.put(schema=bodhi.server.schemas.UpdateCriticalPackages,
+                       permission='edit', renderer='json',
+                       error_handler=bodhi.server.services.errors.json_handler,
+                       validators=(colander_body_validator, validate_release, validate_packages)
+                       )
+def add_critpath_packages(request):
+    """
+    Add a list of critical path packages to a specific release.
+
+    Args:
+        request (pyramid.request): The current request.
+    Returns:
+        dict: The dictionary {'status': 'success'}.
+    """
+    data = request.validated
+    release = data['release']
```

Comment:
Let's move the release into the URL instead of the PUT data. This will allow us to add a GET method.